typings/vade_sidetree.d.ts

```diff
@@ -0,0 +1,262 @@
+/*
+  Copyright (c) 2018-present evan GmbH.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+export type Extensible = Record<string, any>;
+
+/**
+ * Defines the result of a DID resolution operation.
+ *
+ * @see {@link Resolvable.resolve}
+ * @see {@link https://www.w3.org/TR/did-core/#did-resolution}
+ */
+export interface DIDResolutionResult {
+  '@context'?: 'https://w3id.org/did-resolution/v1' | string | string[];
+  didResolutionMetadata: DIDResolutionMetadata;
```

Comment:
type DIDResolutionMetadata is missing  ? Getting error 

=> Cannot find name 'DIDResolutionMetadata'.ts(2304)

Reply:
removed, because its unused